api/swagger-spec/v1.json

```diff
@@ -8882,14 +9026,6 @@
         "allowMultiple": false
        },
        {
-        "type": "string",
```

Comment:
your change should not remove anything from spec, right? if yes, probably you need a rebase and generate specs again.